integration/kubernetes/k8s-attach-handlers.bats

```diff
@@ -20,6 +20,8 @@ setup() {
 }
 
 @test "Running with postStart and preStop handlers" {
+	timeout=30s
```

Comment:
how about defining this in `"${BATS_TEST_DIRNAME}/../../.ci/lib.sh"` or `"${BATS_TEST_DIRNAME}/../../lib/common.bash"` ?